src/codeu/chat/common/RawController.java

```diff
@@ -36,6 +36,8 @@
   // use, the call will fail and null will be returned.
   User newUser(Uuid id, String name, Time creationTime);
 
+  User newNickname(Uuid id, String nickname, Time creationTime);
```

Comment:
i do need time parameter otherwise it won't compile because same method with same parameters